numpy/core/tests/test_simd.py

```diff
@@ -578,6 +578,23 @@ def test_conversion_boolean(self):
         true_vsfx = from_boolean(true_vb)
         assert false_vsfx != true_vsfx
 
+    def test_conversion_expand(self):
+        if self.sfx == "u8":
+            totype = "u16"
+        elif self.sfx == "u16":
+            totype = "u32"
+        else:
+            return
+        data = self._data()
+        expand = getattr(self.npyv, "expand_%s_%s" % (totype, self.sfx))
+        vdata = self.load(data)
+        edata = expand(vdata)
+        # lower half part
+        data_lo = data[:self.nlanes//2]
+        # higher half part
+        data_hi = data[self.nlanes//2:]
+        assert edata == (data_lo, data_hi)
```

Comment:
```suggestion
def test_conversion_expand(self):
        if self.sfx not in ("u8", "u16"):
            return
        totype = int(self.sfx[1:])*2
        expand = getattr(self.npyv, f"expand_{totype}_{self.sfx}")
        # close enough from the edge to detect any deviation
        data   = self._data(self._int_max() - self.nlanes)
        vdata  = self.load(data)
        edata  = expand(vdata)
        # lower half part
        data_lo = data[:self.nlanes//2]
        # higher half part
        data_hi = data[self.nlanes//2:]
        assert edata == (data_lo, data_hi)
```
make the test data close from the max int to detect any deviation